tests/Unit/CssInlinerTest.php

```diff
@@ -3215,6 +3215,9 @@ public function copyUninlinableCssToStyleNodeHasNoSideEffects()
         $expectedHtml = $subject->render();
 
         $styleElement = $domDocument->getElementsByTagName('style')->item(0);
+
+        self::assertInstanceOf(\DOMElement::class, $styleElement);
+
```

Comment:
I don't think we need the empty lines around this addition, which is very closely related to both the preceding and following statements.

Reply:
resolved in 164d2f6